login-fire-social.html

```diff
@@ -66,24 +67,26 @@
       }
     </style>
 
-    <template is="dom-if" if="[[onlySignOut]]">
+
+
+    <template is="dom-if" if="[[_showOneButton]]">
```

Comment:
The element does not have the property `_showOneButton` since you disabled the `login-fire-common-behavior.html` file import.